includes/admin/class-wc-rest-stripe-account-keys-controller.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * Class WC_REST_Stripe_Account_Keys_Controller
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for saving Stripe's test/live account keys. This includes Live Publishable Key,
+ * Live Secret Key, Webhook Secret.
+ */
+class WC_REST_Stripe_Account_Keys_Controller extends WP_REST_Controller {
+	const STRIPE_GATEWAY_SETTINGS_OPTION_NAME = 'woocommerce_stripe_settings';
+
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc/v3';
+
+	/**
+	 * Endpoint path.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'wc_stripe/account_keys';
+
+	/**
+	 * Verify access to request.
+	 */
+	public function check_permission() {
+		return current_user_can( 'manage_woocommerce' );
+	}
+
+	private function get_stripe_settings() {
+		return get_option( self::STRIPE_GATEWAY_SETTINGS_OPTION_NAME, [] );
+	}
```

Comment:
Great idea to put this in a constant and extract a method! Could we move it to a different class so other classes can benefit from it too? E.g. `WC_Stripe_Payment_Gateway` or whatever class would work best?

Reply:
mm I think I might remove this function since it is serving as a wrapping function at this point.

Reply:
I like the idea of moving the constant out into some constants file. Then change the entire plugin to use `STRIPE_GATEWAY_SETTINGS_OPTION_NAME` instead of `woocommerce_stripe_settings`. But it seems like the existing pattern in the codebase is to use `woocommerce_stripe_settings` directly as the option name. I would prefer to keep this in the class until another class needs it, then we can refactor?

Reply:
Sounds good!